src/Models/AngelType.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Query\Builder as QueryBuilder;
+
+/**
+ * @property int     $id
+ * @property string  $name
+ * @property string  $description
+ * @property string  $contact_name
+ * @property string  $contact_dect
+ * @property string  $contact_email
+ * @property boolean $restricted # If users need an introduction
+ * @property boolean $requires_driver_license # If users must have a driver license
+ * @property boolean $no_self_signup # Users can sign up for shifts
+ * @property boolean $show_on_dashboard # Show on public dashboard
+ * @property boolean $hide_register # Hide from registration page
+ *
+ * @method static QueryBuilder|AngelType[] whereId($value)
+ * @method static QueryBuilder|AngelType[] whereName($value)
+ * @method static QueryBuilder|AngelType[] whereDescription($value)
+ * @method static QueryBuilder|AngelType[] whereContactName($value)
+ * @method static QueryBuilder|AngelType[] whereContactDect($value)
+ * @method static QueryBuilder|AngelType[] whereContactEmail($value)
+ * @method static QueryBuilder|AngelType[] whereRestricted($value)
+ * @method static QueryBuilder|AngelType[] whereRequiresDriverLicense($value)
+ * @method static QueryBuilder|AngelType[] whereNoSelfSignup($value)
+ * @method static QueryBuilder|AngelType[] whereShowOnDashboard($value)
+ * @method static QueryBuilder|AngelType[] whereHideRegister($value)
+ */
+class AngelType extends BaseModel
```

Comment:
If angel types should be ordered by name by default we should probably set it on the model instead of every query

Something like

```
protected static function boot(): void
{
    parent::boot();
    static::addGlobalScope('order', fn (Builder $builder) => $builder->orderBy('name');
}
```